src/test/java/study/methodmatcher/SayPrefixMethodMatcher.java

```diff
@@ -0,0 +1,15 @@
+package study.methodmatcher;
+
+import java.lang.reflect.Method;
+
+public class SayPrefixMethodMatcher implements MethodMatcher{
+
+    private static final String TARGET_PREFIX = "say";
+
+    @Override
+    public boolean matches(Method m, Class targetClass, Object[] args) {
+        if (m.getName().startsWith(TARGET_PREFIX))
+            return true;
+        return false;
+    }
```

Comment:
(구현 상의 디테일이라 그냥 넘기셔도 됩니다~)

1.
간단하게 한 줄로 나타낼 수 있을 것 같아요 :)

```suggestion
    @Override
    public boolean matches(Method m, Class targetClass, Object[] args) {
        return (m.getName().startsWith(TARGET_PREFIX));
    }
```